spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RestControllerAdvice.java

```diff
@@ -73,17 +74,23 @@ public VndErrors onException(Exception e) {
 	@ResponseStatus(HttpStatus.CONFLICT)
 	@ResponseBody
 	public VndErrors onConflictException(Exception e) {
+		if (logger.isTraceEnabled()) {
```

Comment:
logging the error first would probably be better ordering, so that one sees the message and then the stack trace.